substrate/frame/system/benchmarking/src/extensions.rs

```diff
@@ -0,0 +1,213 @@
+};
+use sp_runtime::{
+	generic::Era,
+	traits::{AsSystemOriginSigner, DispatchTransaction, Dispatchable, Get},
+};
+use sp_std::prelude::*;
+
+pub struct Pallet<T: Config>(System<T>);
+
+#[benchmarks(where
+	T: Config + Send + Sync,
+    T::RuntimeCall: Dispatchable<Info = DispatchInfo, PostInfo = PostDispatchInfo>,
+	<T::RuntimeCall as Dispatchable>::RuntimeOrigin: AsSystemOriginSigner<T::AccountId> + Clone)
+]
+mod benchmarks {
+	use super::*;
+
+	#[benchmark]
+	fn check_genesis() -> Result<(), BenchmarkError> {
+		let len = 0_usize;
+		let caller = account("caller", 0, 0);
+		let info = DispatchInfo { weight: Weight::zero(), ..Default::default() };
+		let call: T::RuntimeCall = frame_system::Call::remark { remark: vec![] }.into();
+
+		#[block]
+		{
+			CheckGenesis::<T>::new()
+				.test_run(RawOrigin::Signed(caller).into(), &call, &info, len, |_| Ok(().into()))
+				.unwrap()
+				.unwrap();
+		}
+
+		Ok(())
+	}
+
+	#[benchmark]
+	fn check_mortality() -> Result<(), BenchmarkError> {
+		let len = 0_usize;
+		let ext = CheckMortality::<T>::from(Era::mortal(16, 256));
+		let block_number: BlockNumberFor<T> = 17u32.into();
+		System::<T>::set_block_number(block_number);
+		let prev_block: BlockNumberFor<T> = 16u32.into();
+		let default_hash: T::Hash = Default::default();
+		frame_system::BlockHash::<T>::insert(prev_block, default_hash);
+		let caller = account("caller", 0, 0);
+		let info = DispatchInfo {
+			weight: Weight::from_parts(100, 0),
+			class: DispatchClass::Normal,
+			..Default::default()
+		};
+		let call: T::RuntimeCall = frame_system::Call::remark { remark: vec![] }.into();
+
+		#[block]
+		{
+			ext.test_run(RawOrigin::Signed(caller).into(), &call, &info, len, |_| Ok(().into()))
+				.unwrap()
+				.unwrap();
+		}
+		Ok(())
+	}
+
+	#[benchmark]
+	fn check_non_zero_sender() -> Result<(), BenchmarkError> {
+		let len = 0_usize;
+		let ext = CheckNonZeroSender::<T>::new();
+		let caller = account("caller", 0, 0);
+		let info = DispatchInfo { weight: Weight::zero(), ..Default::default() };
+		let call: T::RuntimeCall = frame_system::Call::remark { remark: vec![] }.into();
+
+		#[block]
+		{
+			ext.test_run(RawOrigin::Signed(caller).into(), &call, &info, len, |_| Ok(().into()))
+				.unwrap()
+				.unwrap();
+		}
+		Ok(())
+	}
+
+	#[benchmark]
+	fn check_nonce() -> Result<(), BenchmarkError> {
+		let caller: T::AccountId = account("caller", 0, 0);
+		let mut info = frame_system::AccountInfo::default();
+		info.nonce = 1u32.into();
+		info.providers = 1;
+		let expected_nonce = info.nonce + 1u32.into();
+		frame_system::Account::<T>::insert(caller.clone(), info);
+		let len = 0_usize;
+		let ext = CheckNonce::<T>::from(1u32.into());
+		let info = DispatchInfo { weight: Weight::zero(), ..Default::default() };
+		let call: T::RuntimeCall = frame_system::Call::remark { remark: vec![] }.into();
+
+		#[block]
+		{
+			ext.test_run(RawOrigin::Signed(caller.clone()).into(), &call, &info, len, |_| {
+				Ok(().into())
+			})
```

Comment:
```suggestion
			ext.test_run(RawOrigin::Signed(caller.clone()).into(), &call, &info, len, |_| Ok(().into()))
```

Reply:
My formatter wants to keep it as is. Are you using nightly on your `fmt`?

Reply:
I use the same version as the CI, currently: `cargo +nightly-2023-11-01 fmt`.